### PR TITLE
Created new pom class

### PR DESCRIPTION
**Pull request by Abhishek8954**
please review

### CODE REVIEW

File: com.crm.vtiger1234/src/main/java/com/comcast/crm/objectrepositoryutility/CreatePrroductPage.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package com.comcast.crm.objectrepositoryutility;
+
+public class CreatePrroductPage {
+	
+	//code1
```
Comment (Abhishek8954) [inline com.crm.vtiger1234/src/main/java/com/comcast/crm/objectrepositoryutility/CreatePrroductPage.java:5]: change the xpath 
do not hard code the data

**Review by Abhishek8954 — COMMENTED** [review]
added few comments

File: com.crm.vtiger1234/src/main/java/com/comcast/crm/objectrepositoryutility/CreatePrroductPage.java (hunk 1 above)
Comment (Abhishek8954) [reply com.crm.vtiger1234/src/main/java/com/comcast/crm/objectrepositoryutility/CreatePrroductPage.java:5]: fixed

**Comment by Ankita8596** [comment]
merged

**Pull request merged**